server/circle/client.go

```diff
@@ -0,0 +1,216 @@
+func GetPipelineByID(apiToken string, pipelineID string) (circleci.Pipeline, error) {
+	pl, resp, err := client.PipelineApi.GetPipelineById(getContext(apiToken), pipelineID)
+	resp.Body.Close()
+	return pl, err
+}
+
+// RerunWorkflow reruns a given workflow
+func RerunWorkflow(apiToken string, workflowID string) (circleci.MessageResponse, error) {
+	ms, resp, err := client.WorkflowApi.RerunWorkflow(getContext(apiToken), workflowID, nil)
+	resp.Body.Close()
+	return ms, err
+}
+
+// CancelWorkflow reruns a given workflow
+func CancelWorkflow(apiToken string, workflowID string) (circleci.MessageResponse, error) {
+	ms, resp, err := client.WorkflowApi.CancelWorkflow(getContext(apiToken), workflowID)
+	resp.Body.Close()
+	return ms, err
+}
+
+// GetEnvVarsList returns list of environment variables for given projects
+func GetEnvVarsList(apiToken string, projectSlug string) ([]circleci.EnvironmentVariablePair1, error) {
+	env, resp, err := client.ProjectApi.ListEnvVars(getContext(apiToken), projectSlug)
+	resp.Body.Close()
+	return env.Items, err
+}
+
+// AddEnvVar add an environment variable for given project
+func AddEnvVar(apiToken string, projectSlug string, name string, value string) error {
+	opts := new(circleci.ProjectApiCreateEnvVarOpts)
+	opts.Body = optional.NewInterface(circleci.EnvironmentVariablePair{Name: name, Value: value})
+	_, resp, err := client.ProjectApi.CreateEnvVar(getContext(apiToken), projectSlug, opts)
+	resp.Body.Close()
+	return err
+}
+
+// DelEnvVar delete an environment variable for given project
+func DelEnvVar(apiToken string, projectSlug string, name string) error {
+	_, resp, err := client.ProjectApi.DeleteEnvVar(getContext(apiToken), projectSlug, name)
+	resp.Body.Close()
+	return err
+}
+
+// GetWorkflowMetrics retuirns workflow metrics for a project
+func GetWorkflowMetrics(apiToken string, projectSlug string) ([]circleci.InlineResponse200Items, error) {
+	met, resp, err := client.InsightsApi.GetProjectWorkflowMetrics(getContext(apiToken), projectSlug, nil)
+	resp.Body.Close()
+	return met.Items, err
+}
+
+// GetWorkflowJobsMetrics returns jobs metrics for given workflow
+func GetWorkflowJobsMetrics(apiToken string, projectSlug string, workflowName string) ([]circleci.InlineResponse2002Items, error) {
+	met, resp, err := client.InsightsApi.GetProjectWorkflowJobMetrics(getContext(apiToken), projectSlug, workflowName, nil)
+	resp.Body.Close()
+	return met.Items, err
+}
+
+// ApproveJob approves the pending job in the workflow
+func ApproveJob(apiToken string, approvalRequestID string, workFlowID string) (string, error) {
+	response, resp, err := client.WorkflowApi.ApprovePendingApprovalJobById(getContext(apiToken), approvalRequestID, workFlowID)
+	resp.Body.Close()
+	if err != nil {
+		return "", err
+	}
+
+	return response.Message, nil
+}
+
+// EnvVarExist check if given env var exist
+func EnvVarExist(apiToken string, projectSlug string, name string) (circleci.EnvironmentVariablePair, bool, error) {
```

Comment:
nit: pluralization suggestion. 
```suggestion
// EnvVarExists checks if given env var exists
func EnvVarExists(apiToken string, projectSlug string, name string) (circleci.EnvironmentVariablePair, bool, error) {
```

Reply:
Done :white_check_mark: